### DISCUSSION TITLE
Live Usage

### DISCUSSION THREAD

**Discussion by DemOnJR** [Fabricio-191/valve-server-query#9] [Show and tell]
Hi,
I'm using this for almost 6 months for Counter-Strike 1.6 but i did have some errors with my db cache structure so i needed to rework it and the uptime history is from the last update.
Here you have an live demo of the [valve-server-query](https://github.com/Fabricio-191/valve-server-query) 

- Servers List Example: https://counter-strike-servers.com/servers
- Server Home Example: https://counter-strike-servers.com/server/51.38.60.53:27015
- Server Uptime History: https://counter-strike-servers.com/history/uptime/51.38.60.53:27015
- and more...

In this whole 6 months i did never had an issue with it running every minute of every day.
Keep in mind i'm using an old version from [Fabricio-191](https://github.com/Fabricio-191) because of linux socket things...

**Comment by Fabricio-191** [comment]
That's really nice to hear, i swear there is an update coming, i don't know when xd

**Reply by DemOnJR** [reply]
> That's really nice to hear, i swear there is an update coming, i don't know when xd

CS 2 is also comming this summer <3
